src/index.css

```
@@ -1,4 +1,58 @@
-/* 
-  Le contenu de ce fichier a été déplacé vers la balise <style> dans index.html 
-  pour assurer la compatibilité avec le script Tailwind Play CDN.
-*/
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+@layer base {
+  html {
+    font-smooth: always;
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)